R/04-isomemoAPI.R

```diff
@@ -0,0 +1,86 @@
+callAPI <- function(action, ...) {
```

Comment:
You can rename this file to from "04-isomemoAPI.R" to "03-isomemoAPI.R".
- all helper files with functions start with "03"
- all helper files with modules start with "02"